src/grid/tests/test_cubic.py

```diff
@@ -0,0 +1,392 @@
+r"""Cubic Grid Testing."""
+
```

Comment:
Can you add the Grid Header to this test file?

Reply:
Fixed in 4f0e86c